test-cargo-miri/issue-1705/src/lib.rs

```diff
@@ -0,0 +1,5 @@
+use byteorder::{LittleEndian, ByteOrder};
+
+pub fn use_the_dependency() {
+    let _n = <LittleEndian as ByteOrder>::read_u32(&[1,2,3,4]);
+}
```

Comment:
The new tests duplicate `issue-1567/src/lib.rs`, but I don't know how to write them without duplication (unless `include!("issue-1567/src/lib.rs")` or `#[path = "../../issue-1567/src/lib.rs"] pub mod ...`... Sorry.

Reply:
That's okay, it's not like this is a lot of code.